src/components/Toolbar.jsx

```diff
@@ -32,7 +32,8 @@ function ToolbarLink(props) {
   return <a
     className={classnames('maputnik-toolbar-link', props.className)}
     href={props.href}
-    target={"blank"}
+    rel="noopener noreferrer"
+    target={"_blank"}
```

Comment:
Can this just be `target="_blank"` since it's not pulling from `props`? I tried that locally and it works

Reply:
Yup, fixed :)